src/facebook_example/reaction.clj

```diff
@@ -19,4 +16,19 @@
   [(templates/text-message "Sorry, I didn't get that! :(")])
 
 (defn thank-for-attachment []
-  [(templates/text-message "Thank you for your attachment")])
+  [(templates/text-message "Thank you for your attachment :)")])
+
+(defn help []
+  [(templates/quick-replies-message "What do you need help with?"
+                                    [{:content_type "text"
+                                      :title "Clojure"
+                                      :payload "CLOJURE"}
+                                     {:content_type "text"
+                                      :title "Heroku"
+                                      :payload "HEROKU"}])])
```

Comment:
could use `templates/quick-reply` (https://github.com/prometheus-ai/fb-messenger-clj/blob/master/src/fb_messenger/templates.clj#L48) instead of creating maps.
might be easier for lemmings as don't have to worry about content_type